clothcalc_placeholder.js

```diff
@@ -7799,13 +7807,26 @@
                         eval("MarketWindow.getClearName = " + save.MarketWindow)
                     }
                 };
+
+                _self.injectGetBids = function() {
+		    if (typeof save.MarketWindowTab == "undefined") {
```

Comment:
Just me thinking: injecting this to `showTab` will probably do a lot of unnecessary `getBids` requests, maybe it's enough to do this while the player is creating a new bid? But that may not handle bids that have been canceled or lost.,, Oh, well, Inno can handle a few extra requests. :)